create-entry.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+JEKYLL_ASSETS_DIR=$HOME'/Documents/portfolio/assets/'
+JEKYLL_POSTS_DIR=$HOME'/Documents/portfolio/_posts/'
+
+FOLDER_NAME='';
+TITLE='';
+DETAIL='';
+DESCRIPTION='';
+VID=false;
+IMG=false;
+
+while [ -n "$1" ]; do 
+
+    case "$1" in 
+    -from)
+        dir="$2"
+        echo "Copying files from ${dir} to ${JEKYLL_ASSETS_DIR}"
+        cd ${JEKYLL_POSTS_DIR}
+        FOLDER_NAME=${dir##*/}
+        mkdir ${FOLDER_NAME}
+        cp -rf ${dir} ${JEKYLL_ASSETS_DIR}
+        
+        shift
+        
+        ;;
+    -title)
+        TITLE="$2"
+        shift
+        
+        ;;
+    -detail)
+        DETAIL="$2"
+        shift
+
+        ;;
+    -description)
+       DESCRIPTION="$2"
+       shift
+
+        ;;
+    -img)
+        IMG=true;
+
+        ;;
+
+    -vid)
+       VID=true;
+
+       ;;
+    *) echo "Not valid option"
+      break
+        ;;
+    esac
+
+    shift
+done
+
+TITLE_STRIPPED=${TITLE// /_}
+DATE=`date +%Y-%m-%d`
+TYPE='.md'
+OUTPUT_FILE=${DATE}-${TITLE_STRIPPED}${TYPE}
+
+
+cd ${JEKYLL_POSTS_DIR}
+touch ${OUTPUT_FILE}
+
+echo "Created ${OUTPUT_FILE} in ${JEKYLL_POSTS_DIR}"
+
+echo -n "---
+layout: post
+title: ${TITLE}
+detail: ${DETAIL}
+description: ${DESCRIPTION}" >> ${OUTPUT_FILE}
+
+if $IMG; then
+    echo -n "
+photos:" >> ${OUTPUT_FILE}
+
+    for file in "${JEKYLL_ASSETS_DIR}${FOLDER_NAME}"/*; do
+        f=$(echo "${file##*/}");
```

Comment:
This isn't super portable, but if you're already using bash, you can do something like:

      for file in "${JEKYLL_ASSETS_DIR}${FOLDER_NAME}"/*.@(jpg|jpeg|png|gif)